src/archivist/errors.js

```diff
@@ -1,9 +1,7 @@
 export class InaccessibleContentError extends Error {
   constructor(message) {
-    if (Array.isArray(message)) {
-      message = `\n - ${message.join('\n - ')}`;
-    }
-    super(`The documents cannot be accessed or their contents can not be selected:${message}`);
+    super(`The documents cannot be accessed or their contents can not be selected:${Array.isArray(message) ? `\n - ${message.join('\n - ')}` : message}`);
     this.name = 'InaccessibleContentError';
+    this.reasons = Array.isArray(message) ? message : [message];
```

Comment:
```suggestion
    this.reasons = [].concat(message);
```

Reply:
Oh yes, I'd forgotten about that one, but I was sure there was a simpler way of doing it.